knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-resources.yaml

```diff
@@ -488,9 +488,29 @@ data:
             "includeAll": false,
             "label": "SourceName",
             "multi": false,
-            "name": "source",
+            "name": "nameofSource",
             "options": [],
-            "query": "label_values(kube_pod_labels{label_eventing_knative_dev_sourceName=~\".+\", namespace=\"$namespace\", label_eventing_knative_dev_source=\"$scontroller\"}, label_eventing_knative_dev_sourceName)",
+            "query": "label_values(kube_pod_labels{label_eventing_knative_dev_sourceName=~\".+\", label_eventing_knative_dev_source_name=\"\", namespace=\"$namespace\", label_eventing_knative_dev_source=\"$scontroller\"}, label_eventing_knative_dev_sourceName)",
```

Comment:
I am using an [empty string](https://stackoverflow.com/questions/51293895/how-to-display-all-metrics-that-dont-have-a-specific-label) to indicate that the metric shouldnt exist.